CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to Skipper
+
+## Pull requests only
+
+**DON'T** push to the master branch directly. Always use feature branches and let people discuss changes in pull requests.
+Pull requests should only be merged after all discussions have been concluded and at least 1 reviewer has given their 
+**approval**.
+
+## Guidelines
+
+- **every change** needs a test
+- keep the current code style
```

Comment:
This has been discussed before. If "fixing" go lint means breaking the API, then, yes, this means not to fix go lint. Any other, non-breaking, "go lint fixes" are welcome